Sources/DataPipeline/CustomerIO.swift

```diff
@@ -6,22 +6,15 @@ public extension CustomerIO {
      Call this function when your app launches, before using `CustomerIO.instance`.
      */
     @available(iOSApplicationExtension, unavailable)
-    static func initialize(
-        writeKey: String,
-        logLevel: CioLogLevel = .error,
-        configure configureHandler: ((inout DataPipelineConfigOptions) -> Void)?
-    ) {
-        var cdpConfig = DataPipelineConfigOptions.Factory.create(writeKey: writeKey)
-
-        if let configureHandler = configureHandler {
-            configureHandler(&cdpConfig)
-        }
+    static func initialize(withConfig config: SDKConfigBuilderResult) {
+        // SdkConfig is currently not stored anywhere, we'll later add option to store and access it.
```

Comment:
```suggestion
```

Unless there is a problem caused by not storing the SdkConfig anywhere, I am OK with not storing it somewhere and removing this comment.

Reply:
I've updated the comment, but I want to keep it as a reminder that this decision was made deliberately.